resources/functions/nsfw.yaml

```diff
@@ -0,0 +1,119 @@
+apiVersion: serverless.kyma-project.io/v1alpha1
+kind: Function
+metadata:
+  name: nsfw
+spec:
+  deps: |-
+    { 
+      "name": "nsfw",
+      "version": "1.0.0",
+      "dependencies": {
+        "node-fetch": "2.6.1",
+        "@google-cloud/vision":"2.3.8"
+      }
+    }
+  env:
+    - name: GCP_EMAIL
+      valueFrom:
+        secretKeyRef:
+          key: GCP_EMAIL
+          name: kyma-showcase-secret
+    - name: GCP_API_KEY
+      valueFrom:
+        secretKeyRef:
+          key: GCP_API_KEY
+          name: kyma-showcase-secret
+    - name: API_CONFIG
+      valueFrom:
+        configMapKeyRef:
+          key: config.json
+          name: showcase-functions-config
+  runtime: nodejs14
+  source: |-
+    module.exports = {
+      main: async function (event, context) {
+        async function getBase64(id) {
+          console.log("Getting base64 from DB");
+          const fetch = require('node-fetch');
+          const URL = JSON.parse(process.env.API_CONFIG).API_URL + '/' + id;
+          const response = await fetch(URL);
+          const data = await response.json();
+          const content = data.content;
+          const base64 = content.replace(/data:.*?base64,/, '');
+          console.log("Base64 loaded");
+          return base64;
+        }
+
+        async function getNSFWTag(base64){
+          console.log("Getting NSFW Tag from GCP");
```

Comment:
`Checking NSFW tag in GCP` could be better.

Reply:
In other functions, we follow the same convention.

Reply:
Yes, because in other functions we get informations, in this function we check tags and return true/false info, so I think it's checking, but if you decide getting is better just leave it. 🙌